docs/source/tutorials/overview.rst

```diff
@@ -0,0 +1,28 @@
+API overview
+============
+
+Section which described mental model, flow / pipeline programming,
+messages that carry data, nodes that compute upon them
+
+Device
+######
+
+For more details, click :ref:`here <Device>`
```

Comment:
`:ref: `link to the `Device` page doesn't work correctly. I assume that's because of missing 
```
. toctree::
   :maxdepth: 0
   :hidden:
   :glob:

   device/*
```
in the `device.rst` file. Same comment for the `Pipeline` link below.